messenger.rst

```diff
@@ -1566,6 +1566,9 @@ The transport has a number of options:
 ``delay[exchange_name]`` (default: ``delays``)
     Name of the exchange to be used for the delayed/retried messages
 
+``queues[daily_delay_queues]`` (default: ``false``)
+    Create a delayed queue for each day ``%queue_name_pattern%_%current_day%``.
+    Expires after a day + ttl. Useful for quorum queues, because quorum queues don't support update expire after queue is created.
```

Comment:
```suggestion
    Expires after a day + ttl. Useful for quorum queues, because quorum queues don't support update expire after queue is created.

```